app/docker-compose.yml

```diff
@@ -31,7 +57,10 @@ services:
       - path: ./override.env
         required: false
     depends_on:
-      - app-db
+      app-db:
+        condition: service_healthy
+      # wait_for_phoenix:
+      #   condition: service_completed_successfully
```

Comment:
Uncommenting this causes PR checks to fail, but it's useful to have when running these containers locally.